controller/kubernetes_pod_controller.go

```diff
@@ -29,6 +29,8 @@ import (
 
 const (
 	controllerAgentName = "longhorn-kubernetes-pod-controller"
+
+	delayDuration = 5 * time.Second
```

Comment:
nit: `remountRequestDelayDuration` 

Suggest naming should be clear.

Reply:
Fixed